tools/frr-reload.py

```diff
@@ -1961,6 +1962,44 @@ def compare_context_objects(newconf, running):
     return (lines_to_add, lines_to_del)
 
 
+class LogFmtFormatter(logging.Formatter):
+    def format(self, record):
+        # Escape double quotes in the message, replace newlines with '\n'
+        record.msg = record.getMessage().replace('"', '\\"')
+        record.msg = record.msg.replace('\n', '\\n')
+        # Format the time in RFC 3339 format
+        record.asctime = datetime.datetime.fromtimestamp(record.created, datetime.timezone.utc).astimezone().isoformat(timespec='seconds')
```

Comment:
I don't know what this projects max line length is, but this would be the longest line in the file. Probably something like this to bring it below 120 chars:
```suggestion
        timestamp = datetime.datetime.fromtimestamp(record.created, datetime.timezone.utc)
        record.asctime = timestamp.astimezone().isoformat(timespec='seconds')
```